scripts/check-firmware-revisions.sh

```diff
@@ -3,7 +3,8 @@
 # This is basically copy-paste of 
 #     https://github.com/trezor/trezor-suite/blob/develop/packages/connect-common/scripts/check-firmware-revisions.sh
 #
-#     with small adjustments for. Please keep them in sync if possible.
+#     Small adujstments are made because here we check releases.json here in data/firmware, not those in trezor-suite/connect-common.
```

Comment:
```suggestion
#     Small adjustments are made because here we check releases.json here in data/firmware, not those in trezor-suite/connect-common.
```

Reply:
Fixed.